### PR TITLE
Feature/vehicleloadout

### PR DESCRIPTION
**Pull request by Z1Meme**

### CODE REVIEW

**Comment by Z1Meme** [comment]
@QuentinGruber Ready enough for update, don't close branch though

File: src/servers/ZoneServer2016/zoneserver.ts [hunk 1]
```diff
@@ -3729,19 +3724,43 @@ export class ZoneServer2016 extends EventEmitter {
...
       }
     );
-    /*
+
+    console.log(vehicle._containers)
+    this.initializeContainerList(client, vehicle);
+    
     if(seatId === "0") {
+      const inventory = Object.values(vehicle._containers)[0];
+      console.log("BEGINCHARACTERACCESS")
```
Comment (QuentinGruber) [inline src/servers/ZoneServer2016/zoneserver.ts:3733]:
```suggestion
```

File: src/servers/ZoneServer2016/zoneserver.ts [hunk 2]
```diff
@@ -3729,19 +3724,43 @@ export class ZoneServer2016 extends EventEmitter {
...
+      console.log(inventory)
```
Comment (QuentinGruber) [inline src/servers/ZoneServer2016/zoneserver.ts:3734]:
```suggestion
```

**Review by QuentinGruber — APPROVED** [review]
🚗  Nice 🥳

**Review by QuentinGruber — APPROVED** [review]

**Pull request merged**